src/bucket/test/BucketIndexTests.cpp

```diff
@@ -272,6 +357,177 @@ class BucketIndexTest
         }
     }
 
+    void
+    testLoadContractKeySpanningPages()
+    {
+        size_t page_size = 16384;
+        for (auto const& bucketHash : getBM().getBucketListReferencedBuckets())
```

Comment:
I'm not sure what this for loop is doing, all Buckets either have a page size determined by the cfg or a page size of 0 if they are individually index, you can remove it.